commcare_connect/templates/microplanning/home.html

```
@@ -377,11 +377,12 @@ <h3 class="text-lg font-semibold mb-3">{% translate "Warning" %}</h3>
                         </template>
                     </div>
                     <div class="flex gap-2 justify-between">
-                        <button class="button button-md outline-style" @click="showConfirmModal = false">
+                        <button class="button button-md outline-style" @click="showConfirmModal = false" :disabled="isSaving">
                             {% translate "Back to Edit" %}
                         </button>
-                        <button class="button button-md primary-dark" @click="saveAssignment()">
-                            {% translate "Save and Confirm" %}
+                        <button class="button button-md primary-dark" @click="saveAssignment()" :disabled="isSaving">
+                            <i x-show="isSaving" class="fa-solid fa-spinner fa-spin mr-1"></i>
+                            <span x-text="isSaving ? '{% translate "Saving..." %}' : '{% translate "Save and Confirm" %}'"></span>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)